Gemfile

```diff
@@ -1,2 +1,9 @@
 source 'https://rubygems.org'
-gemspec
\ No newline at end of file
+gemspec
+
+# this is temporary for the feature/plugin-api-2_0 branch and is meant for travis testing
+gem "logstash-core", :path => "./logstash/logstash-core"
```

Comment:
yeah, this is temporary for this refactor branch and will be removed once we are ready to merge back in master. this is just to allow Travis to build on that branch without having to publish ai core gems. see https://github.com/logstash-plugins/logstash-filter-grok/blob/feature/plugin-api-2_0/.travis.yml#L9